docs/platforms/javascript/guides/react/features/react-router.mdx

```diff
@@ -82,6 +82,55 @@ You can instrument [`createMemoryRouter`](https://reactrouter.com/en/main/router
 
 </Alert>
 
+### Custom Error Boundaries
+
+When using `react-router`, errors thrown inside route elements are only re-thrown in **development mode**. In production, these errors won't be surfaced unless manually captured. If you **don't** have a custom error boundary in place, `react-router` will create a default one that "swallows" all errors.
+
+<Alert>
+Note that this only applies to render method and lifecycle errors, since React doesn't need error boundaries to handle errors in event handlers.
+</Alert>
```

Comment:
```suggestion
<Note>
Note, that this only applies to render method and lifecycle errors since React doesn't need error boundaries to handle errors in event handlers.
</Note>
```